crates/egui_demo_lib/src/demo/scrolling.rs

```diff
@@ -292,6 +293,9 @@ impl super::View for ScrollTo {
         ui.horizontal(|ui| {
             scroll_top |= ui.button("Scroll to top").clicked();
             scroll_bottom |= ui.button("Scroll to bottom").clicked();
+            if ui.button("Scroll down by 64px").clicked() {
```

Comment:
It would be nicer demo with two buttons, using ⬆⬇ arrows instead of the text (egui supports those emojis)

Reply:
Done. Like this?

https://github.com/emilk/egui/assets/8009393/1b8663f5-b06d-4a13-a10a-9041fb4f011c